include/stumpless/prival.h

```diff
@@ -0,0 +1,271 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2018-2022 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @file
+ * Severity codes for classifying log entries. See RFC 5424 section 6.2.1 for
+ * details on these values. Severities are defined to be compatible with the
+ * syslog.h header if it is found on the system. Otherwise, they are defined as
+ * closely as possible to the RFC 5424 specification.
+ */
+
+#ifndef __STUMPLESS_PRIVAL_H
+#  define __STUMPLESS_PRIVAL_H
+
+#  include <stumpless/config.h>
```

Comment:
remove all of the severity code from this file, leaving just the new get prival string function declaration.